data/kobold_press/document.json

```diff
@@ -0,0 +1,245 @@
+[
+    {
+        "title": "Kobold Press OGL Content",
+        "slug": "kp",
+        "desc": "Kobold Press Community Use Policy",
+        "license": "Open Gaming License",
+        "author": "Various",
+        "organization": "Kobold Press",
+        "version": "1.0",
+        "copyright": "© Open Design LLC",
```

Comment:
Shouldn't this copyright go to Kobold Press?

Reply:
![image](https://user-images.githubusercontent.com/43607012/227763993-1a6c6868-fd44-4cba-9440-1c7240f70a0b.png)
Open Design LLC seems to be the owner of the Kobold Press name. See footer: https://koboldpress.com

Reply:
The deep magic document.json also has Open Design LLC as the copyright holder, for the same reason mentioned above.
```
"organization": "Kobold Press",
        "version": "1.0",
        "copyright": "Deep Magic for 5th Edition © 2020 Open Design LLC; Authors: Dan Dillon, Chris Harris, and Jeff Lee."
```